app/database/decoder/breeze_buddy/template.py

```
@@ -3,14 +3,116 @@
+    """Convert old-format voice fields to the new unified voice_config.
+
+    Handles templates stored with the old schema:
+      - tts_voice_name: "rhea" / "mira" / "sara"
+      - mira_voice_id: str
+      - cartesia_voice_configurations: {...}
+      - elevenlabs_voice_configurations: {...}
+
+    Transforms them into voice_config + voice_config_overrides.
+    Already-migrated templates (with voice_config) pass through unchanged.
+    """
+    if "voice_config" in data and data["voice_config"]:
```

Ruff (0.15.6)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)